frontend/vue/components/UserAccount/SyllabusCard.vue

```diff
@@ -35,8 +38,23 @@ export default defineComponent({
     syllabus: {
       type: Object,
       required: false,
+      default: () => { }
+    },
+    userId: {
+      type: String,
+      required: true,
       default: ''
```

Comment:
Same here.. if prop is required do we need to pass a default?

Reply:
fixed via https://github.com/Qiskit/platypus/pull/1660/commits/72ad42a1e8c137150ac9cfbf7500fd724333acd0